lldb/source/Core/Value.cpp

```diff
@@ -120,6 +120,19 @@ AddressType Value::GetValueAddressType() const {
   return eAddressTypeInvalid;
 }
 
+Value::ValueType Value::GetValueTypeFromAddressType(AddressType address_type) {
+  switch (address_type) {
+    case eAddressTypeFile:
+      return Value::ValueType::FileAddress;
```

Comment:
Is there a 100% overlap between AddressType and ValueType? If yes, it might be good to unify them upstream.

Reply:
If there is 100% overlap, it makes me wonder why both are needed.

Reply:
There's not 100% overlap, `Value::ValueType` has an extra `Scalar` case, which `AddressType` doesn't have.

I'm also going to open a patch with this commit before merging this pull request by the way.